src/AzureSqlAction.ts

```diff
@@ -90,15 +88,50 @@ export default class AzureSqlAction {
 
     private async _executeSqlFile(inputs: ISqlActionInputs) {
         core.debug('Begin executing sql script');
-        let scriptContents: string;
-        try {
-            scriptContents = fs.readFileSync(inputs.sqlFile, "utf8");
+
+        // sqlcmd should be added to PATH already, we just need to see if need to add ".exe" for Windows
+        let sqlCmdPath: string;
+        switch (process.platform) {
```

Comment:
will process.platform always be lowercase or should there be .`toLowerCase()` just in case?

Reply:
They should be always lowercase according to the docs: https://nodejs.org/api/process.html#processplatform